.github/workflows/test.yml

```diff
@@ -14,9 +14,9 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-10.15
-          - ubuntu-latest
-          - windows-latest
+          - "macos-11"
+          - "ubuntu-22.04"
+          - "windows-2022"
```

Comment:
Too bad we have to type this list of three strings into two files.  Perhaps we could at least leave a comment in both places directing maintainers to update the other list any time they are going to update one of them.

Reply:
Left notes in both .yml files.

There's some documentation about reusing workflows, and I'm pretty sure that wasn't a thing when we originally added GA workflows to zfec: https://docs.github.com/en/actions/using-workflows/reusing-workflows

I don't know if "reusing workflows" means `strategy` portion of both workflows can be refactored to someplace common, or if there's something else that is relevant.  Maybe not yet.